gensim/models/phrases.py

```diff
@@ -106,17 +206,23 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
         `delimiter` is the glue character used to join collocation tokens, and
         should be a byte string (e.g. b'_').
 
+        `average_sentence_size`: expected number of tokens in average sentence.
+            max_vocab_size / average_sentence_size sentences is used to build python
+            dictionary per thread.
+            Lowering `average_sentence_size` will increase memory demands.
         """
         if min_count <= 0:
-            raise ValueError("min_count should be at least 1")
-
+            raise ValueError("min_count should be 1")
+        if min_count > 1:
+            logger.warning("min_count should be 1")
```

Comment:
The only reason I left it is not to break API (it has limited functionality to subtract this count from all frequencies before counting final bi-gram score).
I would be grateful for example how to solve the parameter removal elegantly.